censys/asm/inventory.py

```diff
@@ -31,17 +32,32 @@ def search(
         Returns:
             dict: Inventory search results.
         """
+        if workspaces is None:
+            w: List[str] = [self.get_workspace_id()]
```

Comment:
I think this should just be `workspaces` instead of `w`

Reply:
if I rename it to `workspaces` it yells at me because it's already defined, but if I change it to `workspaces = [self.get_workspace_id()]`, it is now defined as `list[str]` instead of `List[str]`, which causes a type error

Reply:
Huh those should be the same type 🤔